CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
```

Comment:
We can remove the "unreleased" header.

Reply:
ok; and add it back in the `post-release` stuff